packages/trader/src/sass/app/modules/contract/digits.scss

```diff
@@ -3,19 +3,27 @@
 /** @define cq-bottom-ui-widgets; weak */
 .cq-bottom-ui-widgets {
     z-index: 4;
+    overflow: visible;
+    height: 0;
+    top: unset !important;
+    bottom: 80px !important;
 
     .bottom-widgets {
         left: -3.5em;
     }
 }
 
-/** @define digits */
+/** @define digits; weak */
 .digits {
     display: flex;
     align-items: center;
     position: relative;
     margin: 0 2.5em 0 1em;
 
+    /* postcss-bem-linter: ignore */
```

Comment:
if you're ignoring this, do you still need weak on the top?

Reply:
Good point, I'm not sure why the `weak` property didn't work. I'll try to see if I can get it to pass without using `bem-linter: ignore`.